src/Microsoft.Health.Dicom.Core/Extensions/DicomMediatorExtensions.cs

```diff
@@ -252,4 +252,15 @@ public static class DicomMediatorExtensions
         EnsureArg.IsNotNull(parameters, nameof(parameters));
         return mediator.Send(new QueryWorkitemResourceRequest(parameters), cancellationToken);
     }
+
+    public static Task<RetrieveWorkitemResponse> RetrieveWorkitemAsync(
+        this IMediator mediator,
+        string workitemInstanceUid,
+        CancellationToken cancellationToken = default)
+    {
+        EnsureArg.IsNotNull(mediator, nameof(mediator));
+        EnsureArg.IsNotNullOrWhiteSpace(workitemInstanceUid, nameof(workitemInstanceUid));
```

Comment:
I might have misguided you before haha. I believe, `IsNotEmptyOrWhiteSpace` would be better here.